### DISCUSSION TITLE
Global autoimport of utils

### DISCUSSION THREAD

**Discussion by victj99** [nuxt/nuxt#22676] [Questions]
Hi, I have a question
Is there a recomended way to have global autoimports of utils scripts

Is this good idea? 
![image](https://github.com/nuxt/nuxt/assets/44973221/ca16c7d9-a2ed-4e19-b6db-ea50c73814f6)

Thanks.

**Comment by TheAlexLichter** [comment]
Hey 👋🏻
The separation of `utils` / and `server/utils` being available only in the "vue"-part/"nitro"-part of your app is intentional.
If you re-use utilities across both sides, you could instead create a file and re-export them (e.g. create a `server/utils/date.utils.ts` and re-export the `addDaysToDate` function there)
